test/integration/server/inspec/server_spec.rb

```diff
@@ -46,9 +49,11 @@
 
 describe mysql_conf('/etc/my.cnf') do
   its('mysqld.log_bin_trust_function_creators') { should eq '1' }
+  its('mysqld.innodb_large_prefix') { should eq 'true' } if os.release.to_i < 8 # Deprecated in mysql 5.7
   its('mysqld.userstat') { should eq 'true' }
+  its('mysqld.innodb_file_format') { should eq 'barracuda' } if os.release.to_i < 8 # Deprecated in mysql 5.7
```

Comment:
I removed these in #85, please revert this change.

Reply:
oop